Tools/Replay/LR_MsgHandler.cpp

```diff
@@ -195,7 +195,10 @@ void LR_MsgHandler_GPA_Base::update_from_msg_gpa(uint8_t gps_offset, uint8_t *ms
     require_field(msg, "TimeUS", time_us);
     wait_timestamp_usec(time_us);
 
-    uint16_t vdop, hacc, vacc, sacc;
+    uint16_t vdop = 0;
+    uint16_t hacc = 0;
+    uint16_t vacc = 0;
+    uint16_t sacc = 0;
```

Comment:
I believe @lucasdemarchi has already correct this - in a different way - in master